### DISCUSSION TITLE
Mobile image and video compression

### DISCUSSION THREAD

**Discussion by ferologics** [MobileNativeFoundation/discussions#69] [General]
Hi, I want to ask the community's opinion on optimal image and video compression strategies. 

I am not very knowledgable on this front but for example, what `JPEG` data compression quality do you use? Or should you use `PNG` almost exclusively? What are the criteria when deciding to compress multimedia? When (not) to opt in for compression? Should avatar images have same compression quality as regular user images sent in the chat? Or should they differ?

If someone with more experience in this matter can chime in with real world examples that would be very much appreciated! :)

**Comment by ghost** [comment]
As a guide I would use PNG for anything drawn by designers with lots of blocks of single colour, and JPEG for any natural camera-taken image.  I would do an experiment to see for yourself on JPEG compressions.  File size drops dramatically when you move from the highest quality and may be a good compromise.

**Reply by ferologics** [reply]
That's actually a very sensible measure! Although, I prefer to use vector (`PDF`) format images for designer assets over `PNG`s.
